config/initializers/rack/attack.rb

```diff
@@ -27,19 +27,25 @@ class Attack
     end
 
     throttle("site_hits", limit: 40, period: 2) do |request|
-      track_and_return_ip(request.env["HTTP_FASTLY_CLIENT_IP"])
+      track_and_return_ip(request)
     end
 
     throttle("tag_throttle", limit: 2, period: 1) do |request|
       if tag_request?(request)
-        track_and_return_ip(request.env["HTTP_FASTLY_CLIENT_IP"])
+        track_and_return_ip(request)
       end
     end
 
-    def self.track_and_return_ip(ip_address)
+    def self.track_and_return_ip(req)
+      ip_address = if ApplicationConfig["FASTLY_API_KEY"].present?
+                     req.env["HTTP_FASTLY_CLIENT_IP"]
+                   else
+                     req.remote_ip
+                   end
       return if ip_address.blank?
 
-      Honeycomb.add_field("fastly_client_ip", ip_address)
+      Honeycomb.add_field("fastly_client_ip", req.env["HTTP_FASTLY_CLIENT_IP"])
+      Honeycomb.add_field("remote_ip", req.remote_ip)
```

Comment:
This is for my own testing/logging in production. I would like to see if remote_ip can yield similar result to `HTTP_FASTLY_CLIENT_IP`